### PR TITLE
Windows installation from sources documentation

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: docs/rst/developer_manual/installation/sources/windows.rst [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    :local:
+    :backlinks: none
+    :depth: 2
+
+
+Dependencies installation
+=========================
+
+|ddsrouter| depends on *eProsima Fast DDS* library and certain Debian packages.
+This section describes the instructions for installing |ddsrouter| dependencies and requirements in a Linux
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/windows.rst:24]:
```suggestion
This section describes the instructions for installing |ddsrouter| dependencies and requirements in a Windows
```

File: docs/rst/developer_manual/installation/sources/windows.rst [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 2
+
+
+Dependencies installation
+=========================
+
+|ddsrouter| depends on *eProsima Fast DDS* library and certain Debian packages.
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/windows.rst:23]: In general:
I would suggest to create a shortcut for FastDDS as it is for `|ddsrouter|`, so it always keeps the same format

File: docs/rst/developer_manual/installation/sources/windows.rst [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+`Gtest Installation Guide <https://github.com/google/googletest>`_ for a detailed description of the Gtest installation
+process.
+
+
+.. _windows_sources_dependencies:
+
+Dependencies
+------------
+
+|ddsrouter| has the following dependencies, when installed from sources in a Linux environment:
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/windows.rst:132]:
```suggestion
|ddsrouter| has the following dependencies, when installed from sources in a Windows environment:
```

**Review by jparisu — CHANGES_REQUESTED** [review]
Small changes

File: docs/rst/developer_manual/installation/sources/windows.rst (hunk 2 above)
Comment (rsanchez15) [reply docs/rst/developer_manual/installation/sources/windows.rst:23]: I agree but it won't go in this PR

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**